### ISSUE TITLE
Compiler version change from 1.0.0-beta.... to 2.0.0 after qx package migrate 

### ISSUE THREAD

**Issue by rommni** [qooxdoo/qooxdoo-compiler#737]
I tested the last version of compiler (last commit: https://github.com/qooxdoo/qooxdoo-compiler/commit/d144c1498ece2e47cc8708aa1280a9e5448978b5) by doing git clone and npm pack. 

After running npm install with new tar.gz generated i run qx compile and it asked me : 
```
*** Framework and/or compiler dependencies in Manifest need to be updated.
*** Please run 'qx package migrate' to apply the changes. If you don't want this, downgrade to a previous version of the compiler.
```

So I run `npx qx package migrate` and it changed in my Manifest.json the following lines:
```
  -  "@qooxdoo/framework": "^6.0.0-beta-20190617-1533",
  -  "@qooxdoo/compiler": "^1.0.0-beta.20190618-0639",
  +  "@qooxdoo/framework": "^6.0.0-beta-20200609-0430",
  +  "@qooxdoo/compiler": "^2.0.0",
```

The version of compiler is changed and so at next compile I have several message of qx package which doesn't support v2 of the compiler.

**Comment by hkollmann** [comment]
Have you run qx pkg update && qx pkg upgrade?

**Comment by rommni** [comment]
No only qx package migrate like asked in the output of qx compile. Do you want me to test with update and upgrade to see the result ?

**Comment by rommni** [comment]
Fixed by #739
